### PR TITLE
Show label from last revision

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Generate review display name using latest revision instead of first revision

### CODE REVIEW

**Review by pakrym — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs [hunk 1]
```diff
@@ -75,7 +75,7 @@ public string DisplayName
         public DateTime LastUpdated => Revisions.LastOrDefault()?.CreationDate ?? CreationDate;
 
         [JsonIgnore]
-        public string Language => Revisions.FirstOrDefault()?.Files.FirstOrDefault()?.Language;
+        public string Language => Revisions.LastOrDefault()?.Files.LastOrDefault()?.Language;
```
Comment (pakrym) [inline src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs:78]: Is using the last file important?

**Comment by JoshLove-msft** [comment]
Out of curiosity was this requested by someone? We did this intentionally so that the review could be labeled independent of each revision.

**Review by JoshLove-msft — APPROVED** [review]

File: src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs (hunk 1 above)
Comment (praveenkuttappan) [reply src/dotnet/APIView/APIViewWeb/Models/ReviewModel.cs:78]: There were few python reviews which initially updated as json and then created new revision with wheel. In general I think its better to refer latest value even though most of the reviews will have same value for first and last revision.

**Comment by praveenkuttappan** [comment]
@JoshLove-msft : With automatic API reviews, It's not correct when display name is showing oldest label and it's confusing too.

**Pull request merged**